multihost_test/bz_automation/test_chkpwd-onlyroot.py

```diff
@@ -156,3 +157,45 @@ def test_19810(self, multihost, bkp_pam_config):
         (result, result1, exit_status) = ssh1.execute_cmd('su - local_anuj', stdin="password123")
         client.run_command("faillock")
         assert "Password: su: Authentication failure" not in result1.readlines()[0]
+
+    def test_16727(self, multihost, bkp_pam_config, create_localusers):
+        """
+        :title: PAM can't identify the user when running from external host
+        :id: ae78e4d6-ba95-11ee-8362-845cf3eff344
+        :bugzilla: https://issues.redhat.com/browse/RHEL-16727
+        :steps:
+          1. Create "testuser" user and set password
```

Comment:
I don't see any code for this step

Reply:
This is with fixture create_localusers

Reply:
You are right. I always forget about the fixtures :facepalm: